app/views/insured/families/find_sep.html.erb

```diff
@@ -6,21 +6,49 @@
   <div>
     <div class="mb-4">
       <h1><%= l10n("insured.families.special_enrollment_period") %></h1>
-      <p class="mb-4"><%= l10n("insured.families.final_sep_experienced_any_events") %></p>
-      <h2><%= l10n("insured.top_life_changes") %></h2>
-      <% if @qualifying_life_events %>
+      <% if EnrollRegistry.feature_enabled?(:qle_commonality_threshold) %>
+        <p class="mb-4"><%= l10n('insured.families.specual_enrollment_period.disclaimer') %></p>
+        <% def list_class(qles) 'two-column' if qles.length > 5 end %>
+        <section>
+          <h2><%= l10n('insured.families.special_enrollment_period.common_header') %></h2>
+          <ul class=<%= list_class(@common_seps) %>>
+            <% @common_seps.each_with_index do |qle, index| %>
+              <li class="<%=pundit_class Family, :updateable?%> mb-3"><%= qle_link_generator(qle, index) %></li>
+            <% end %>
+          </ul>
+        </section>
+        <% if @rare_seps.present? %>
+          <button id="showMoreLink" class="button secondary my-3"><%= l10n('insured.families.special_enrollment_period.show_rare') %></button>
+          <section class="hidden mt-3" id="showLifeChanges">
+            <h2><%= l10n('insured.families.special_enrollment_period.rare_header') %></h2>
+            <ul class=<%= list_class(@rare_seps) %>>
+              <% @rare_seps.each_with_index do |qle, index| %>
+                <li class="<%=pundit_class Family, :updateable?%> mb-3"><%= qle_link_generator(qle, index) %></li>
+              <% end %>
+            </ul>
+            <div class="align-content-center my-4">
+              <label class="weight-n">
+                <input type="checkbox" id="no_qle_checkbox" data-toggle="modal" data-target="#outside-open-enrollment" name="no_qualifying_event" value="true">
+                <%= l10n("insured.families.final_sep_none_of_the_situations_listed") %>
+              </label>
+            </div>
+          </section>
+        <% end %>
+      <% else %>
```

Comment:
originally I kept the whole `@qualifying_life_events` check which wrapped around the new registry check.

I ended up removing it - it was copied from legacy, and I'm not sure what it was doing. I know what it was _trying_ to do, but empty arrays are considered truthy in ruby, so that check never failed.